crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -65,6 +67,18 @@ pub struct Room {
     own_user_id: OwnedUserId,
     inner: Arc<SyncRwLock<RoomInfo>>,
     store: Arc<DynStateStore>,
+
+    /// The most recent few encrypted events. When the keys come through to
+    /// decrypt these, the most recent relevant one will replace
+    /// `latest_event`. (We can't tell which one is relevant until
+    /// they are decrypted.)
```

Comment:
I'm curious, why don't we know?

Reply:
Because we don't know the event type.

Reply:
Ah, of course. It could be bot notice or whatever.